lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -48,9 +48,27 @@ class HttpApiEvents {
       },
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8021
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'httpApi', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            method: { type: 'string' },
+            path: { type: 'string' },
+            authorizer: {
+              type: 'object',
```

Comment:
It can be also `string`, see: https://github.com/serverless/serverless/blob/01a03d103bde0476690ab2e61c81a9f00ae607df/lib/plugins/aws/package/compile/events/httpApi/index.js#L375-L376